packages/manifest-schemas/formats/web3api.plugin/0.0.1-prealpha.1.json

```diff
@@ -9,27 +9,33 @@
   ],
   "properties": {
     "format": {
+      "description": "Polywrap plugin YAML format version.",
       "type": "string",
       "const": "0.0.1-prealpha.1"
     },
     "language": {
+      "description" : "Plugin language.",
       "type": "string",
       "format": "pluginLanguage"
     },
     "schema": {
+      "description": "Path to graphql schema.",
       "type": "string",
       "format": "file"
     },
     "import_redirects": {
+      "description": "Redirects enabling the import of plugins.",
```

Comment:
Redirects source URI to local wrapper or plugin.

Reply:
Updated.